.github/bin/free-disk-space.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+set -euo pipefail
+
+function free_up_disk_space_ubuntu()
+{
+    local packages='azure-cli aspnetcore-* dotnet-* ghc-* firefox*
+        google-chrome-* google-cloud-* libmono-* llvm-* imagemagick
+        postgresql-* rubu-* spinxsearch unixodbc-dev mercurial esl-erlang
+        microsoft-edge-stable mono-* msbuild mysql-server-core-*
+        php-* php7* powershell* mongodb* microsoft-edge* subversion ansible'
```

Comment:
bash doesn't have multiline strings. So when you apply sed to prepend `~n` you'll get different values on different lines.

either convert to array or remove leading whitespace. if you keep the linebreaks then use `tr` to remove them before feeding to `sed`.